File: program176.java
/*
OUTPUT:

Enter the number:
5
Given number is:5
1       *       2       *       3       *       4       *       5       *


Enter the number:
-4
Given number is:-4
1       *       2       *       3       *       4       *

*/

import java.lang.*;
import java.util.*;

class Pattern
{
	private int iNo;
	
	public void Accept()
	{
		Scanner sobj=new Scanner(System.in);
		
		System.out.println("Enter the number:");
		iNo=sobj.nextInt();
		
		
	}
	
	public void Display()
	{
		System.out.println("Given number is:"+iNo);
		
	}
	
	public void DisplayPattern()
	{
		int iCnt=0;
		if(iNo<0)
		{
			iNo=-iNo;
		}
		
		for(iCnt=1;iCnt<=iNo;iCnt++)
		{
			System.out.print(iCnt+"\t"+"*"+"\t");
		}
		
	}
	
}

class program176
{
	public static void main(String arg[])
	{
		Pattern pobj=new Pattern();
		
		pobj.Accept();
		pobj.Display();
		pobj.DisplayPattern();
		
	}
}

File: program176.c
/*
Enter the number:
5
*       *       *       *       *

*/
#include<stdio.h>

void Display(int iNo)
{
	static int iCnt=0;
	if(iCnt!=iNo)
	{
		printf("*\t");
		iCnt++;
		Display(iNo);
	}
	
	
}

int main()
{
	int iValue=0;
	printf("Enter the number:\n");
	scanf("%d",&iValue);
	
	Display(iValue);
	
	return 0;
}